code/modules/surgery/organs/wings.dm

```diff
@@ -145,9 +145,61 @@
 	name = "pair of bee wings"
 	desc = "A pair of bee wings. They seem tiny and undergrown"
 	icon_state = "beewings"
-	flight_level = WINGS_FLIGHTLESS
+	flight_level = WINGS_COSMETIC
+	actions_types = list(/datum/action/item_action/organ_action/use/bee_dash)
 	wing_type = "Bee"
 
+/datum/action/item_action/organ_action/use/bee_dash
+	var/jumpdistance = 3
+	var/jumpspeed = 1
+	var/recharging_rate = 100
+	var/recharging_time = 0
+
+/datum/action/item_action/organ_action/use/bee_dash/Trigger()
+	var/mob/living/carbon/L = owner
+
+	if(L.stat != CONSCIOUS || L.buckling) // Has to be concious and unbuckled
```

Comment:
Check for restraint, being able to dash while cuffed is going to be horrible to play against as security